### ISSUE TITLE
Metrics (LCP, FCP) are not the same from lighthouse report and original trace

### ISSUE THREAD

**Issue by micajuine-ho** [GoogleChrome/lighthouse#12399]
#### Provide the steps to reproduce
1. Run LH on https://www.abc-s.dk/ disabling cache on Network cache and no CPU or network throttling through DevTools
2. After the report is created, click on original trace within the report
3. Notice how the LCP and FCP timings are different from those on the report

![image](https://user-images.githubusercontent.com/30711678/115908722-60ce3400-a41f-11eb-90c3-39088d43e2bd.png)
![image (1)](https://user-images.githubusercontent.com/30711678/115908757-6c215f80-a41f-11eb-81d1-e191bbaabfdf.png)

#### What is the current behavior?
LCP and FCP in the trace (~700ms) are not the same as reported in the LH report (2.9s)

#### What is the expected behavior?
LCP and FCP in the trace to be the same as the reported values

#### Environment Information
* Affected Channels: Extension
* Lighthouse version:
* Chrome version: 90.0.4430.85 (Official Build)
* Node.js version:
* Operating System:

**Related issues**
/cc @kristoferbaxter

**Comment by adamraine** [comment]
This is expected: https://github.com/GoogleChrome/lighthouse/blob/master/docs/throttling.md#devtools-lighthouse-panel-throttling

TL;DR if you need the metrics to match the trace, uncheck "Simulated throttling" in the Lighthouse settings.
